### DISCUSSION TITLE
dial tcp 127.0.0.1:25: connect: connection refused

### DISCUSSION THREAD

**Discussion by benstaf** [postalserver/postal#3092] [Help with using Postal] [answered]
When I try to connect to Postal from an app hosted on the same server, I get the error:

`dial tcp 127.0.0.1:25: connect: connection refused`

Postal works normally if I connect from an app hosted on separate server

It's a [common issue](https://www.redswitches.com/blog/localhost-refused-to-connect/#WhylocalhostrefusedtoconnectErrorHappens) , but I don't know how to address it in this case

**Comment by benstaf** [comment, marked as answer]
I connected to `172.17.0.1` and it worked for me. I found that on [Stackoverflow](https://stackoverflow.com/a/48547074)

**Reply by willpower232** [reply]
If this is your answer, it suggests that you aren't using host networking as set up by default in Postal  and may have other issues sending or receiving email over SMTP.

**Reply by benstaf** [reply]
I set up Postal with Caddy like in the [Documentation](https://docs.postalserver.io/getting-started/installation) (Docker), except that I had to reinstall Caddy to make both web apps work on the same server 

If other issues happen, I will post them, but for now I am quite happy it works
